cmd/collector/Dockerfile

```diff
@@ -1,5 +1,11 @@
+FROM alpine as certs
```

Comment:
Do we want this to be `apline:latest`, or pinned to a version?

Which version of `alpine` does this command use currently?

Reply:
why does it matter? It's a throw-away layer anyway

Reply:
Because it makes it harder to debug if/when it fails. 
Why use an arbitrary version of a dependency when we can control it?

Reply:
if we pin it, we also pin the version of certificates, which is worse imo

Reply:
I don't understand your argument. 

Are you saying that it is better to use an arbitrary version instead of using `apline:latest`, or pinning to a version (as I mentioned earlier)?

Or are you saying that the command `FROM alpine as certs` doesn't pull in an arbitrary version and gets the latest certificates?